modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/tests

```diff
@@ -495,7 +495,27 @@
       cli_args = 'Mesh/gen/ring_radii="2 3" Mesh/gen/ring_intervals="2 2"
                   Mesh/gen/ring_block_names="ring1"
                   --mesh-only "poly_2d_err.e"'
-      expect_err = 'This parameter must have the appropriate size if it is set.'
+      expect_err = 'This parameter must have the appropriate size if it is provided.'
+      detail = "ring_block_names size is not consistent with number of rings in the mesh."
+    []
+    [err_ring_block_ids_quad]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/ring_radii="2 3" Mesh/gen/ring_intervals="2 2"
+                  Mesh/gen/ring_block_ids="20"
+                  Mesh/gen/quad_center_elements="true"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must have the appropriate size if it is provided.'
+      detail = "ring_block_ids size is not consistent with number of rings in the mesh."
```

Comment:
Make the detail different from each other to avert the CIVET error.

Reply:
I edited the detail. Thank you.